diff --git a/pipenv/project.py b/pipenv/project.py
index b96487c10..c8babd85a 100644
--- a/pipenv/project.py
+++ b/pipenv/project.py
@@ -69,7 +69,7 @@ from pipenv.utils.shell import (
     system_which,
 )
 from pipenv.utils.toml import cleanup_toml, convert_toml_outline_tables
-from pipenv.vendor import click, plette, tomlkit
+from pipenv.vendor import plette
 
 try:
     # this is only in Python3.8 and later
