diff --git a/tests/unit/test_utils.py b/tests/unit/test_utils.py
index c8a177e97..6e7779520 100644
--- a/tests/unit/test_utils.py
+++ b/tests/unit/test_utils.py
@@ -238,7 +238,7 @@ class TestUtils:
     ):
         def mock_version(path):
             return version_output.split()[1]
-        monkeypatch.setattr("pipenv.vendor.pythonfinder.utils.get_python_version", mock_version)
+        monkeypatch.setattr("pythonfinder.utils.get_python_version", mock_version)
         assert dependencies.python_version("some/path") == version
 
     @pytest.mark.utils
