From 0e2ca997b421febcda662cad9b7dffe043df4a03 Mon Sep 17 00:00:00 2001
From: Oz Tiram <oz.tiram@gmail.com>
Date: Thu, 21 Sep 2023 15:30:09 +0200
Subject: [PATCH] fix import of plette

---
 pipenv/project.py | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/pipenv/project.py b/pipenv/project.py
index b96487c10..dbb3d775e 100644
--- a/pipenv/project.py
+++ b/pipenv/project.py
@@ -23,6 +23,9 @@ except ImportError:
 
 import contextlib
 
+import click
+import tomlkit
+
 from pipenv.cmdparse import Script
 from pipenv.environment import Environment
 from pipenv.environments import Setting, is_in_virtualenv, normalize_pipfile_path
@@ -69,7 +72,7 @@ from pipenv.utils.shell import (
     system_which,
 )
 from pipenv.utils.toml import cleanup_toml, convert_toml_outline_tables
-from pipenv.vendor import click, plette, tomlkit
+from pipenv.vendor import plette
 
 try:
     # this is only in Python3.8 and later
-- 
2.41.0

